public/index.html

```diff
@@ -6,17 +6,39 @@
     <link
       rel="stylesheet"
       href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
-    />
+      />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Make an appointment to get your COVID vaccine"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+      content="Make an appointment to get your Massachusetts COVID vaccine"
```

Comment:
Make an appointment to get your COVID vaccine in Massachusetts (for consistency)

Reply:
Fixed